editors/vscode/src/browser.ts

```diff
@@ -83,6 +83,16 @@ function startClient(context: vscode.ExtensionContext) {
                         return;
                     },
                 );
+                client.onRequest("slint/preview_message", async (msg: any) => {
+                    if (previewPanel) {
+                        // map urls to webview URL
+                        if (msg.command === "highlight") {
+                            msg.data.path = previewPanel.webview.asWebviewUri(Uri.parse(msg.data.path)).toString();
+                        }
```

Comment:
It would we nice if we could just forward the message as-is without having to look into it.  But unfortunately, the preview and the LSP don't use the same URL scheme.  (the webview uses something like `https://file+.vscode-resource.vscode-cdn.net/home/..../foo.slint`)